src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static int index = 0;
+    private static String separator = "-------------------------------------------";
```

Comment:
Separator here acts like a magic constant. Maybe consider declaring it final and making it capitalized?